docs/articles/new-expensify/connections/Uber-for-Business.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Uber for Business
+description: Learn how to connect Uber for Business to your Expensify workspace, invite employees, and automate receipt collection for Uber Rides and Uber Eats.
+keywords: [New Expensify, Uber for Business, Uber integration, automate Uber receipts, connect Uber to Expensify, U4B, Uber Eats, Uber Rides, receipt automation]
+---
+
+<div id="new-expensify" markdown="1">
+
+Connect your Expensify workspace to Uber for Business to automatically import Uber Rides and Uber Eats receipts for your team. This guide walks you through setup, employee management, and receipt import.
+
+# Connect the Uber for Business Integration
+
+## Step 1: Enable Receipt Partners in your Workspace
+1. Go to **Workspaces > [Workspace Name] > More features**.
+2. Toggle on **Receipt partners**.
+3. Once enabled, a new **Receipt partners** section will appear in the left-hand menu.
+
+## Step 2: Connect Uber for Business
+1. Go to **Workspaces > [Workspace Name] > Receipt partners**
+3. Click **Connect** next to Uber for Business.
+4. Authorize or create your Uber for Business (U4B) account in the new browser tab that opens.
+5. When complete, the tab will close, and a list of workspace members not yet invited to U4B will appear.
+6. Confirm which workspace members should be invited to use the connection (they’ll all be selected by default).
```

Comment:
```suggestion
6. Confirm which workspace members should be invited to U4B (they’ll all be selected by default).
```

I feel like this is clearer as to what happens.

Reply:
Maybe also something about that then triggering an invite for them to accept from Uber?